MonstieWash/Assets/Scripts/PlayerHand.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Playerhand : MonoBehaviour
+{
+    //Unity Inspector
+	[SerializeField]
+    [Range(1.0f, 50.0f)]
+	private float cursorSpeed = 20;
+
+    //Private
+    private Transform m_Transform;
+    private Vector3 handPosition;
+	private float moveHorizontal;
+	private float moveVertical;
+	
+    // Start is called before the first frame update
+    void Start()
+    {
+        m_Transform = gameObject.transform;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        handPosition = Camera.main.WorldToScreenPoint(m_Transform.position);
+        moveHorizontal = Input.GetAxis("Horizontal");
+		moveVertical = Input.GetAxis("Vertical");
```

Comment:
Just realizing this, but we'll have to pivot to the new input system at some point to allow for controller support. For now though, this is okay - just know that it's temporary. We can worry about that at a later date.